src/apis/shops.ts

```
@@ -0,0 +1,12 @@
+import { instance } from './axios';
+
+const shops = '/shops';
+
+export const wishlistHandler = async () => {
+  try {
+    const response = await instance.get(`${shops}/liked`)
+    return response
+  } catch (err) {
+    throw err;
```

Biome (1.9.4)

[error] The catch clause that only rethrows the original error is useless.

An unnecessary catch clause can be confusing.
Unsafe fix: Remove the try/catch clause.


(lint/complexity/noUselessCatch)